config/default/common/config/metadata/layers/olci/sentinel-3a/S3A_OLCI_Chlorophyll_a.md

```diff
@@ -0,0 +1,5 @@
+The Chlorophyll *a* layer provides the near-surface concentration of chlorophyll *a* in milligrams of chlorophyll pigment per cubic meter (mg/m<sup>3</sup>) in the ocean.
+
+The Chlorophyll *a* product is available from the Ocean and Land Colour Instrument (OCLI) aboard the European Space Agency's (ESA) Sentinel-3A and Sentinel-3B satellites. The sensor and imagery resolution is 1 km, and the temporal resolution is daily.
+
+References: doi:[10.5067/S3A/OLCI/L2/ERR/OC/2022](https://doi.org/10.5067/S3A/OLCI/L2/ERR/OC/2022)
```

Comment:
This DOI link is Not Found.

Reply:
I did notice that. I'll have to ask the data provider why it isn't working.